Libraries/CO.CDP.Authentication.Tests/ClaimServiceTests.cs

```diff
@@ -22,15 +22,49 @@ public void GetUserUrn_ShouldReturnUrn_WhenUserHasSubClaim()
     [Fact]
     public void GetUserUrn_ShouldReturnNull_WhenUserHasNoSubClaim()
     {
-        var claims = new List<Claim>();
-        var httpContextAccessor = GivenHttpContextWith(claims);
+        var httpContextAccessor = GivenHttpContextWith([]);
 
         var claimService = new ClaimService(httpContextAccessor.Object);
         var result = claimService.GetUserUrn();
 
         result.Should().BeNull();
     }
 
+    [Fact]
+    public void HaveAccessToOrganisation_ShouldReturnFalse_WhenUserHasNoTenClaim()
+    {
+        var httpContextAccessor = GivenHttpContextWith([]);
+
+        var claimService = new ClaimService(httpContextAccessor.Object);
+        var result = claimService.HaveAccessToOrganisation(Guid.NewGuid());
+
+        result.Should().BeFalse();
+    }
+
+    [Fact]
+    public void HaveAccessToOrganisation_ShouldReturnFalse_WhenDoesNotHaveAccessToOrganisation()
+    {
+        var tenValue = "H4sIAAAAAAAAA42QS2/CMBCE/0rlM87DJGD71Fa0KAeoSkKrtkLI2E5wIXZwEh5C/Pc64lCkXqo97czqm9WcwbyWFtAzmFsNKGitprngtDB7r91QFCBEh4ds9zGePL8ei3aTpVWaoLERk4Mkyz0aLd93avZ5TL+zx1MCemDKSulAozWz5d2btCVz4lPJ1NapolOlFpbtOwej+6JzPG5KcOmBTGqmmxrQrzNIhLuPh4LnEeYQkzCAURhgSHgeQo5IP44wZgGOfzMzWTdue7EF06pmjTL6hkUGggd5P4Y8iAmMBCaQhDmDbLBisYzwIGToD2tmtrJjgLStqq1yVS16rirlLtZNU9XU97moPM5WSsvG5Lni0ruW55vbP/x/xqfcVNfEh9EkmYLFpZsfgDXIV6cBAAA=";
```

Comment:
I'd prefer to have in a more readable format so it's clear the organisationId is part of the value, but I understand it wasn't easy/feasable?

It might also be a problem to the next person who comes to maintain this test.

Reply:
One way is to create/build the token, let me see what I can do